app/javascript/components/search/controls/KeywordSearch.js

```diff
@@ -22,15 +22,21 @@ export default function KeywordSearch({ keywordPrompt }) {
   const [showTextSearchHelpModal, setTextShowSearchHelpModal] = useState(false)
 
   const textSearchModalContent = (<div>
-    <h4 className="text-center">Title and description search</h4><br/>
-    Using the search box below will perform a text-based search of all study titles and descriptions, and will return
-    those that contain any of the specified terms. You can quote (&quot;) phrases to find matches that contain the entire
-    phrase, and you can combine both single terms and quoted phrases in your search.
+    <h4 className="text-center">Title, author, and description search</h4>
+    <p>Use the seach box below to perform a text-based search on study titles, decriptions and authors*.</p>
+    <ul>
+      <li>A search without (&quot;)s will return any study that contains any of the specified terms.</li>
+      <li>To search for an exact phrase within a study title or description use (&quot;)s around the entire phrase like:</li>
+      <p>&quot;single cell&quot;</p>
+      <li> To search for studies that contain an exact phrase and/or the other search term/s, combine single terms and quoted phrase like:</li>
+      <p>&quot;single cell&quot; Smith</p>
+    </ul>
+    <p>*Structured data for authors is new in SCP, and many studies lack it, so author search results may be limited.</p>
```

Comment:
I'd omit the asterisk, but if we keep it let's space it properly.

```suggestion
    <p>* Structured data for authors is new in SCP, and many studies lack it, so author search results may be limited.</p>
```